examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -385,6 +387,15 @@ void PairingCommand::OnCommissioningComplete(NodeId nodeId, CHIP_ERROR err)
     }
     else
     {
+        if (mDeviceIsICD)
+        {
+            CHIP_ERROR deleteEntryError =
+                GetICDClientStorage().DeleteEntry(chip::ScopedNodeId(mNodeId, CurrentCommissioner().GetFabricIndex()));
```

Comment:
Why are we deleting things here?  At the very least, that should be documented.

Reply:
when commissioning fails with error, we need to remove clientInfo from storage. update the comment here, thanks